src/routes/-components/RepositoriesSection.tsx

```diff
@@ -0,0 +1,164 @@
+      </h2>
+      <p className="text-sm/base mt-[10px] text-center text-[#C6C6C6]/80">
+        Use advanced filters to find a project you love and make your first
+        commit message.
+      </p>
+
+      <div className="mx-auto mt-[30px] w-full rounded-[12px] border border-[#294740] bg-[#383C3A]/20 px-5 py-10 lg:max-w-[1256px] lg:px-6 lg:py-[55px]">
+        <div className="mx-auto max-w-[1030px]">
+          <div className="mx-auto flex w-full max-w-[690px] items-center gap-3 rounded-[12px] border border-[#14A97C]/80 bg-[#8C9E99]/15 px-5 py-4 md:px-6">
+            <svg
+              className="size-4 shrink-0"
+              viewBox="0 0 17 17"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M15.4531 15.5L10.7865 10.8333M12.342 6.94444C12.342 9.95133 9.90445 12.3889 6.89757 12.3889C3.89069 12.3889 1.45312 9.95133 1.45312 6.94444C1.45312 3.93756 3.89069 1.5 6.89757 1.5C9.90445 1.5 12.342 3.93756 12.342 6.94444Z"
+                stroke="#989A9A"
+                strokeWidth="1.67"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+            </svg>
+
+            <input
+              type="text"
+              className="flex-1 bg-transparent text-sm/[18px] text-[#989A9A] focus:outline-none"
+              placeholder="Search project"
+              onChange={(e) => {
+                const value = e.target.value.trim();
+                if (value !== searchTerm) {
+                  setSearchTerm(value);
+                }
+              }}
+            />
+          </div>
+
+          <div className="mt-5">
+            <Swiper
+              slidesPerView={"auto"}
+              spaceBetween={12}
+              rewind={false}
+              loop={false}
+            >
+              {filterLanguages.map((lang) => (
+                <SwiperSlide key={lang} className="!w-max">
+                  <button
+                    className={cn(
+                      "min-w-[40px] whitespace-nowrap rounded-full border px-2.5 py-1 font-ff-poppins text-sm text-white",
+                      lang === selectedLanguage
+                        ? "border-[#02FBB0]/80 bg-[#635D5D]"
+                        : "border-[#f3faf8]/80",
+                    )}
+                    onClick={() => {
+                      if (lang === selectedLanguage) return;
+                      setSelectedLanguage(lang);
+                    }}
+                  >
+                    {lang}
+                  </button>
+                </SwiperSlide>
+              ))}
+            </Swiper>
+          </div>
+
+          <div className="mt-[30px] grid gap-x-8 gap-y-[30px] md:grid-cols-2 lg:grid-cols-3">
+            {searchedProjects.map((project) => (
+              <div className="relative isolate overflow-hidden rounded-[12px] border border-[#294740]/80 bg-[#413535]/[.53] px-4 pb-5 pt-7">
+                <svg
```

Comment:
let's try and centralize all the inlined svg ocons inside the src/components/wrappers/custom-icons.tsx file